znai-reactjs/package.json

```diff
@@ -18,6 +18,7 @@
     "katex": "^0.10.0-rc.1",
     "lunr": "2.0.3",
     "mark.js": "github:julmot/mark.js",
+    "mermaid": "^8.9.2",
```

Comment:
did you by any chance check how heavy the js lib is? I.e. difference between final js bundle before and after this lib? 
At some point I need to figure out how to include them dynamically based on what is being used in the docs. Not the issue for enterprise on-prem docs, but could be an issue for open source docs

Reply:
Seemingly adds nothing (a little suspicious)...

Before:

```File sizes after gzip:

  254.39 KB  build/static/js/2.49a2d6d7.chunk.js
  42.97 KB   build/static/js/main.5c546696.chunk.js
  11.14 KB   build/static/css/main.f925a2e6.chunk.css
  10.53 KB   build/static/css/2.94918964.chunk.css
  736 B      build/static/js/runtime-main.0d152442.js```

After:

```File sizes after gzip:

  254.7 KB  build/static/js/2.796555c6.chunk.js
  43.02 KB  build/static/js/main.3c1dd85f.chunk.js
  11.14 KB  build/static/css/main.19641413.chunk.css
  10.53 KB  build/static/css/2.94918964.chunk.css
  736 B     build/static/js/runtime-main.0d152442.js```

Reply:
That could be a wrong `before`. Could you please run build on the branch without mermaid if you have time?

Reply:
Yeah, I ran from a clean clone vs. this one (and checked compiled output for the string "mermaid"), looks like it's pretty low overhead from what's already bundled.